ra_aid/agents/ciayn_agent.py

```
@@ -187,6 +263,10 @@ def _estimate_tokens(content: Optional[Union[str, BaseMessage]]) -> int:
             text = content.content
         else:
             text = content
+
+        # create-react-agent tool calls can be lists
+        if isinstance(text, List):
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)